formats/finder_email_signup/publisher/details.json

```diff
@@ -9,7 +9,7 @@
   ],
   "properties": {
     "beta": {
-      "type": "boolean"
+      "$ref": "#/definitions/finder_beta"
```

Comment:
Here's the presenter, which I think will be fine:
https://github.com/alphagov/finder-frontend/blob/master/app/presenters/signup_presenter.rb#L14-L16

Interestingly finders themselves have already stopped using this:
https://github.com/alphagov/finder-frontend/blob/master/app/presenters/finder_presenter.rb#L31-L33